css/tools/w3ctestlib/Sources.py

```diff
@@ -3,6 +3,7 @@
 # Initial code by fantasai, joint copyright 2010 W3C and Microsoft
 # Licensed under BSD 3-Clause: <http://www.w3.org/Consortium/Legal/2008/03-bsd-license>
 
+from __future__ import print_function
```

Comment:
Defer to @gsnedders if this needs to land in https://github.com/gsnedders/w3ctestlib-git or somewhere else.

Reply:
Doesn't need to land anywhere else.